src/routes/accomodationRoutes.js

```diff
@@ -1,18 +1,27 @@
 import express from "express";
-import * as validate from '../middlewares/validateInputs';
+import * as valid from '../middlewares/validateInputs';
 import * as Accomodation from '../controllers/accomodationController';
 import isLoggedIn from "../middlewares/authenticate"
-import { adminUser} from '../middlewares/authorize';
+import { adminUser, isRequester} from '../middlewares/authorize';
+import validate from '../middlewares/validate';
+import { ratingSchema } from "../validations/ratingValidation";
+import { addRating, allRatings, getRatings } from '../controllers/ratingController';
 
 const accomodationRouter=express.Router();
-accomodationRouter.post('/v1/accommodations/register',isLoggedIn, adminUser, validate.addAccomodation,Accomodation.newEntry);
+accomodationRouter.post('/v1/accommodations/register',isLoggedIn, adminUser, valid.addAccomodation,Accomodation.newEntry);
 
 accomodationRouter.get('/v1/accommodations',isLoggedIn, adminUser, Accomodation.getAll);
 
 accomodationRouter.get('/v1/accommodations/:Id',isLoggedIn, adminUser,Accomodation.getById);
 
-accomodationRouter.patch('/v1/accommodations/update/:Id', isLoggedIn, adminUser,validate.updateAccomodation,Accomodation.doUpdate);
+accomodationRouter.patch('/v1/accommodations/update/:Id', isLoggedIn, adminUser,valid.updateAccomodation,Accomodation.doUpdate);
 
 accomodationRouter.delete('/v1/accommodations/remove/:Id',isLoggedIn, adminUser,Accomodation.doDelete);
 
+accomodationRouter.post('/v1/accommodations/:accomodationId/rating', isLoggedIn, isRequester, validate(ratingSchema), addRating);
```

Comment:
It is better to create a rating router file on its own

Reply:
done with it recheck